thrift/admin.thrift

```diff
@@ -179,12 +179,22 @@ service AdminService {
   * RefreshWorkflowTasks refreshes all tasks of a workflow
   **/
   void RefreshWorkflowTasks(1: shared.RefreshWorkflowTasksRequest request)
-   throws (
+    throws (
       1: shared.BadRequestError badRequestError,
       2: shared.DomainNotActiveError domainNotActiveError,
       3: shared.ServiceBusyError serviceBusyError,
       4: shared.EntityNotExistsError entityNotExistError,
-   )
+    )
+
+  /**
+  * ResendReplicationTasks sends replication tasks from remote cluster
```

Comment:
**to** remote cluster?

Reply:
Updated the comment to request replication tasks from remote cluster and apply task to current cluster